apl-core/src/test/java/com/apollocurrency/aplwallet/apl/testutil/DbManipulator.java

```diff
@@ -32,8 +32,17 @@ public DbManipulator(Path dbFile) {
 
     public DbManipulator(DbProperties dbProperties, PropertiesHolder propertiesHolder) {
         Objects.requireNonNull(dbProperties, "dbProperties is NULL");
-        this.databaseManager = new DatabaseManagerImpl(dbProperties, propertiesHolder == null ? new PropertiesHolder() : propertiesHolder);
-        this.populator = new DbPopulator(databaseManager.getDataSource(), "db/schema.sql", "db/data.sql");
+        PropertiesHolder propertiesHolderParam = propertiesHolder == null ? new PropertiesHolder() : propertiesHolder;
+        this.databaseManager = new DatabaseManagerImpl(dbProperties, propertiesHolderParam);
+
+        String dataScriptPath = "db/data.sql";
```

Comment:
Why we require such code here?

Reply:
It's possible to skip test data for manual testing in 'spacial case' when 'skip parameter' is specified explicitly.

Reply:
Please remove this. Simply add option to Constructor `schemaOnly` if required.

Reply:
I didn't want change current behavior, so it works as before. The **data.sql** is loaded by default.
It's possible to do not load it by supplying "apl.testData = false" explicitly when PropertiesHolder exists and contains it.